homeassistant/components/alexa/entities.py

```diff
@@ -386,7 +386,7 @@ class CoverCapabilities(AlexaEntity):
     def default_display_categories(self):
         """Return the display categories for this entity."""
         device_class = self.entity.attributes.get(ATTR_DEVICE_CLASS)
-        if device_class == cover.DEVICE_CLASS_GARAGE:
+        if device_class in (cover.DEVICE_CLASS_GARAGE, cover.DEVICE_CLASS_GATE):
```

Comment:
`DEVICE_CLASS_GATE` needs to be checked for in the `interfaces()` method below. 

```python
        if device_class not in (cover.DEVICE_CLASS_GARAGE, cover.DEVICE_CLASS_GATE):
            yield AlexaPowerController(self.entity)
```

Per Alexa API Guidelines anything in the `GARAGE` display category, should only yield the `Alexa.ModeController` to enable the Voice by PIN feature.

Reply:
Thanks.  I'll clean this up tomorrow after I have some sleep.

Reply:
Should be good to go now. Added a test to be sure.